config/default.conf

```diff
@@ -204,6 +204,7 @@ dpc-in-data:			xml
 
 alter-statement:			obsolete
 comment-paragraphs:			obsolete
+control-division:			obsolete
```

Comment:
as it is very old and kind of "exotic" - use "unconformable" here and "+obsolete" in lax.inc